e2e/fixtures/basic/doc/index.mdx

```diff
@@ -1,5 +1,7 @@
 # Hello World
 
+### -22222222
```

Comment:
It is a bit weird to include this case in the `basic` fixture. We can create another fixture for this case.

Reply:
And we should using multiple `-1111111111` to test, thus TOC will generate with `-1111111111` and `-1111111111-1`

```
## -1111111111

## -1111111111
```